content/en/docs/test.md

```diff
@@ -65,10 +65,10 @@ culpa qui officia deserunt mollit anim id est laborum.
   - To make a sub-item, indent two spaces.
     - This is a sub-sub-item.
```

Comment:
@bep Any idea why this list item is not rendered at the correct level? It is rendered with its parent.

Reply:
It is hard to have any opinion on a question like that looking at a GitHub diff.

Reply:
https://deploy-preview-8622--kubernetes-io-master-staging.netlify.com/docs/test/#lists